app/views/scenarios/_scenario.html.erb

```diff
@@ -0,0 +1,14 @@
+<div id="scenario-<%= scenario.id %>" class="flex-center-container">
+  <% if scenario.completed == true %>
+    <i class="completed-icon far fa-check-square"></i>
+  <% else %>
+    <i class="completed-icon far fa-square"></i>
+  <% end %>
+  <p style="padding: 0 15px; margin-bottom: 0"><%= scenario.name %></p>
+  <%= link_to  edit_campaign_scenario_path(@campaign, scenario) do %>
+    <i class="fas fa-pencil-alt link-icon"></i>
+  <% end %>
+  <%= button_to "Like this post", campaign_scenario_path(@campaign, scenario), method: :put,
```

Comment:
remove "Like this post" from `button_to` and include `button_to` on the checkbox icon